tests/testthat/test-convGDX2mif.R

```diff
@@ -92,10 +94,13 @@ test_that("Test if REMIND reporting is produced as it should and check data inte
   # gdxPaths <- c(gdxPaths, .findAMTgdx())
 
   numberOfMifs <- 0
+  computedVariables <- NULL
+
   for (gdxPath in gdxPaths) {
     numberOfMifs <- numberOfMifs + 1
     message("Running convGDX2MIF(", gdxPath, ")...")
     mifContent <- convGDX2MIF(gdxPath, gdx_ref = gdxPath)
+    computedVariables <- unique(c(computedVariables, getItems(mifContent, dim = 3.3)))
```

Comment:
As a first step, this will check all the mif files generated from gdx files for available REMIND variables in remind2 reporting. 

We might have to add additional gdx files, if the existing ones don't cover all the configurations to be run when running reporting for specific projects.

Reply:
Also, this is not a specific testing where a project gdx is compared to the REMIND variables of the corresponding project template (not sure if this is necessary).

Reply:
I think that is a great start. Still thinking whether it may be worth adding a parameter  to `getREMINDTemplateVariables()` such as `type = standalone` or `REMIND-MAgPIE` or however this should be called.

Reply:
Don't know enough about coupled runs and reporting to understand what difference it makes? 

Would we need two tests, one for standalone run with a standalone gdx, and one for a REMIND-MAgPIE with a gdx representing a coupled run?

Reply:
Coupled runs simply have much more variables related to land-use, agriculture etc. that they get from MAgPIE. But they are not part of the REMIND gdx file, but are attached to the mif file later, [in reporting.R](https://github.com/remindmodel/remind/blob/3b719d8dd4856b709a793cb34b9e0b35082244ff/scripts/output/single/reporting.R#L86-L103). Therefore, I don't think a remind2 test can cover these, but rather the MAgPIE people should set up tests for that, I guess.